compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -297,6 +298,12 @@ def container_id(self, c_id: UUID_LIKE_T | None):
     def _fn_cache_key(self, fn: t.Callable):
         return fn, self.container_id
 
+    def get_metadata(self) -> dict:
+        """Return metadata relevant to the current Executor instance
+        (e.g., Python version).
+        """
+        return {"python_version": sys.version, "sdk_version": __version__}
```

Comment:
sys.version returns something like: 
'3.10.4 (main, Jul  6 2022, 10:10:25) [Clang 13.1.6 (clang-1316.0.21.2.5)]' which is fairly long.  I used the first 3 of sys.version_info ie. '3.10.4'  (see get_env_details() in utils/__init__.py).  The shorter one is easier to compare at the far end which was my reasoning.  as we are not comparing yet, I supposed the whole string is ok (and should be small in comparison to function size).  Fine with both, just highlighting it.

Reply:
That's a good call. I used a slightly different methodology to get the version, but same idea.